UI/Web/src/app/book-reader/_components/book-reader/book-reader.component.ts

```diff
@@ -275,6 +275,8 @@ export class BookReaderComponent implements OnInit, AfterViewInit, OnDestroy {
 
   writingStyle: WritingStyle = WritingStyle.Horizontal;
 
+  pointer: boolean = false;
```

Comment:
Add a JSDoc on top to indicate this controls if the cursor is a pointer or not based on if over a pagination area or not.